### PR TITLE
An array `package.json#source` should build specified entries

### PR DESCRIPTION
**Pull request by AceTheCreator**
# ↪️ Pull Request
Running parcel build my-package/ should build a.html, b.html if my-package/package.json contains: 
`{
  "source": ["a.html", "b.html"]
}`
#5387

### CODE REVIEW

**Comment by mischnic** [comment]
Please add an integration test

File: packages/core/core/src/requests/EntryRequest.js [hunk 1]
```diff
@@ -122,6 +122,40 @@ class EntryResolver {
           entries: [{filePath: source, packagePath: entry}],
           files: [{filePath: pkg.filePath}],
         };
+      } else if (pkg && Array.isArray(pkg.source)) {
+        let source;
+        const entries = [];
```
Comment (wbinnssmith) [inline packages/core/core/src/requests/EntryRequest.js:127]: nit: `let entries = [];`, even though it's not reassigned this is a style thing in Parcel (though I personally also would prefer const)

**Review by wbinnssmith — COMMENTED** [review]
Looks good! As @mischnic mentioned, this should have an integration test.

File: packages/core/core/src/requests/EntryRequest.js (hunk 1 above)
Comment (alphabet-codes) [reply packages/core/core/src/requests/EntryRequest.js:127]: Ok. Thanks for the review

Comment (alphabet-codes) [reply packages/core/core/src/requests/EntryRequest.js]: @wbinnssmith I have submitted an updated version of this pull request, kindly review it thanks.

Comment (wbinnssmith) [reply packages/core/core/src/requests/EntryRequest.js]: looks good! Mind writing an integration test that exercises this code? Should be good to go from there 😄

Comment (alphabet-codes) [reply packages/core/core/src/requests/EntryRequest.js]: @wbinnssmith I'll do that later today.

**Comment by mischnic** [comment]
@AceTheCreator Would be great if you could add some tests so that we can merge this!

**Comment by AceTheCreator** [comment]
> @AceTheCreator Would be great if you could add some tests so that we can merge this!

I'll do that later today @mischnic

**Comment by AceTheCreator** [comment]
> > @AceTheCreator Would be great if you could add some tests so that we can merge this!
> 
> I'll do that later today @mischnic



> @AceTheCreator Would be great if you could add some tests so that we can merge this!

**Comment by AceTheCreator** [comment]
@mischnic I'm having an issue writing test for this. Any idea on how I can do it?

**Comment by mischnic** [comment]
Add a new folder to `packages/core/integration-tests/test/integration/` containing multiple source files and a package.json with a `source` field. Similar to `packages/core/integration-tests/test/integration/html-pkg-source` but with an array instead.

And then run that test in `packages/core/integration-tests/test/html.js`.

**Pull request closed without merging**